drafts/str4d-stratum/draft1.rst

```diff
@@ -0,0 +1,256 @@
+Abstract
+========
+
+This ZIP describes the Zcash variant of the Stratum protocol, used by miners to
+communicate with mining pool servers.
+
+
+Specification
+=============
+
+``mining.subscribe()``
+----------------------
+
+Request::
+
+    {"id": 1, "method": "mining.subscribe", "params": ["CONNECT_HOST", CONNECT_PORT, "MINER_USER_AGENT", "SESSION_ID"]}\n
+
+- ``CONNECT_HOST`` (str): The host string used by the miner.
+
+  - ``pool.example.com``
+
+- ``CONNECT_PORT`` (int): The port used by the miner.
+
+  - ``3337``
+
+- ``MINER_USER_AGENT`` (str): A free-form string specifying the type and version
+  of the mining software. Recommended syntax is the User Agent format used by
+  Zcash nodes.
+
+  - ``Zatoshi/1.0.0``
```

Comment:
I doubt Zcash will be a Stratum-enabled mining client anytime soon, won't be? If @str4d works on miner called Zatoshi, then it is completely fine.